README.rst

```diff
@@ -51,11 +51,11 @@ If you would like to test your image's integration with the Civis Platform local
 4. Run the container: ``docker run --rm -p 8888:8888 --env-file my.env test``.
 5. Access the notebook at the ip of your Docker host with port 8888 (e.g., ``http://localhost:8888/notebooks/notebook.ipynb``).
 
-Integration Testing Code Changes with the Civis Platform
---------------------------------------------------------
+Integration Testing Code Changes with Civis Platform
+----------------------------------------------------
 
 The scripts ``tests/build_dev_image.sh`` and ``tests/run_dev_image.sh`` can be used to test the
-integration of code changes with the Civis Platform.
+integration of code changes with Civis Platform.
 
 From the top directory in the repo type::
 
```

Comment:
I tested the changes with the test instructions in this section, and I was able to see the notebook locally at `http://localhost:8888/notebooks/notebook.ipynb`.